packages/trader/src/App/Containers/Layout/app-contents.jsx

```diff
@@ -1,6 +1,6 @@
 import classNames           from 'classnames';
 import PropTypes            from 'prop-types';
-import React                from 'react';
+import React from 'react';
```

Comment:
why?

Reply:
last time i forgot to disable my auto save reformat 😅 .  let me revert this line